src/components/_card.scss

```
@@ -172,7 +172,7 @@ $sb-thumbnail-desc-color: $sb-color-light-1;
 $sb-thumbnail-title-color: $sb-color-dark-2;
 $sb-thumbnail-title-active-color: $sb-color-blue-2;
 
-$sb-thumbnail-image-list: mood nps opentext responses rating;
+$sb-thumbnail-image-list: mood nps opentext responses rating locked;
```

```suggestion
$sb-thumbnail-image-list: mood nps opentext responses rating lock;
```
or `coming-soon` something like

---

When it comes to iconography, I would just pick a name that represents what the icon is - the context in which it is used should only matter to naming if it is named within that context.

---

so "lock" or "coming soon"?

---

lock 👍